### ISSUE TITLE
Is there any criterion for choosing alpha?

### ISSUE THREAD

**Issue by LyuYuxin** [pseulki/IB-Loss#1]
I wish to adopt IB loss to YOLOX for object detection....
  However after several epoch finetuing the mAP drop from 80% to 30%.  
  I choose 1000 for alpha. When calculating the grad, I use the channel values of every pixel point on the input feature map of the classfier pred as "feature vector".

**Comment by pseulki** [comment]
Hi, thank you for interest in IB loss!
We recommend you to set alpha so that you can get the similar level of loss when you train your model without IB loss.
This is because the gradient norm is different for each dataset or model.
For example, for image classification (CIFAR), I get loss around 3~5 with my training strategy (i.e., learning rate, optimizer,, ).
When I set alpha to 1 and train with IB loss, its loss becomes as small as 0.002 ~ 0.003 (since the loss is divided by norm of the gradient.). That's why we multiply 1000 (=alpha) to get the typical loss so that I can train the model with my original learning scheme. Or, I can change learning strategy, such as changing learning rate. 
In sum, check your loss when you used IB loss, and if it is too big or too small, please set alpha so that you can get the typical loss with your original training setting.
I hope this works for you!

**Comment by LyuYuxin** [comment]
understood! thank you for response.
